src/screens/SurveyDetailScreen.tsx

```diff
@@ -55,11 +57,13 @@ const useSurvey = (id?: string) => {
 
 // eslint-disable-next-line complexity
 export const SurveyDetailScreen = ({ route }: Props) => {
+  const scrollViewRef = useRef<ScrollView>(null);
   const archived = route.params?.archived;
-  const { loading, refetch, survey } = useSurvey(route.params?.id);
+  const surveyId = route.params?.id;
+  const { loading, refetch, survey } = useSurvey(surveyId);
```

Comment:
i didn't check where and why, but pull to refresh causes red screen

<img width="943" alt="Bildschirmfoto 2021-07-07 um 15 34 55" src="https://user-images.githubusercontent.com/1942953/124768403-ee0e2880-df38-11eb-9f40-f476686e834c.png">

Reply:
this seems like the fast refresh bug with refetch
https://github.com/apollographql/apollo-client/issues/6816

can you reproduce it without fast refreshing?

Reply:
no.
thx.
good.